fuzz/fuzz_targets/addrman.rs

```diff
@@ -0,0 +1,39 @@
+#![no_main]
+
+use bitcoin::consensus::encode;
+use bitcoin::p2p::address::AddrV2Message;
+use bitcoin::p2p::ServiceFlags;
+use floresta_wire::address_man::AddressMan;
+use floresta_wire::address_man::LocalAddress;
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &[u8]| {
+    let mut address_man = AddressMan::default();
+    let addrv2_msg_vec: Result<Vec<AddrV2Message>, _> =
```

Comment:
There are some places like here, where the type could be inferred from the RHS. We usually add explicit types if it can be inferred.

Reply:
Done.